app/Template/config/color.php

```diff
@@ -0,0 +1,36 @@
+<div class="page-header">
+    <h2><?= t('Custom colors') ?></h2>
+</div>
+<section>
+<form method="post" action="<?= $this->url->href('config', 'color') ?>" autocomplete="off">
+
+    <?= $this->form->csrf() ?>
+
+    <table class="color-settings">
+    <thead>
+        <tr>
+            <th><?= t('Color') ?></th>
+            <th><?= t('Label') ?></th>
+            <th><?= t('Background') ?></th>
+            <th><?= t('Border') ?></th>
+            <th><?= t('Selectable?') ?></th>
+        </tr>
+    </thead>
+    <tbody>
+    <? foreach ($colors as $color_id => $color) { ?>
```

Comment:
This syntax is not compatible with recent version of PHP, you should use something like:

```php
<?php foreach ($colors as $color_id => $color): ?>
...
<?php endforeach ?>
```